drops/arc_multicloud_connector.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Using the Multicloud Connector Enabled by Azure Arc ",
+  "Summary": "This Drop provides a guide to using the Multicloud connector enabled by Azure Arc to onboard EC2 instances to Azure and view AWS resources in Azure.",
+  "Description": "This Jumpstart Drop describes how to use the Multicloud connector enabled by Azure Arc to onboard EC2 instances to Azure Arc, view an inventory of AWS resources in Azure, and run Azure Resource Graph queries to view resources across clouds.",
+  "Authors": [
+    {
+      "Name": "Mike Rhoads",
+      "Link": "https://www.linkedin.com/in/mike-rhoads/"
+    }
+  ],
+  "Source": "https://github.com/Azure/arc_jumpstart_drops/tree/main/tutorial_guide/arc_multicloud_connector",
+  "Type": "tutorial_guide",
+  "Difficulty": "Medium",
+  "ProgrammingLanguage": [
+    "CloudFormation"
+  ],
+  "Products": [
+    "Arc",
+    "Arc-enabled Servers",
+    "EC2",
+    "AWS",
```

Comment:
As a product, i woudl put AWS EC2 and then you can put AWS as a tag

Reply:
fixed